homeassistant/components/yamaha/media_player.py

```diff
@@ -368,10 +366,27 @@ def play_media(self, media_type, media_id, **kwargs):
         if media_type == "NET RADIO":
             self.receiver.net_radio(media_id)
 
+    async def async_enable_output(self, enabled, port):
+        """Enable or disable an output port.."""
+        await self.hass.async_add_job(self.enable_output, port, enabled)
+
     def enable_output(self, port, enabled):
         """Enable or disable an output port.."""
         self.receiver.enable_output(port, enabled)
 
+    async def async_set_scene(self, scene):
+        """Set the current scene."""
+        await self.hass.async_add_job(self.set_scene, scene)
+
+    @property
+    def scene(self):
```

Comment:
Where is this used?

Reply:
Nowhere, I added it for completeness ...

Reply:
Please remove it.

Reply:
Removed it as my Receiver does not return the last scene but the source.